CHANGELOG.md

```diff
@@ -16,6 +16,10 @@
 ## 3.21.0 (2023-19-01)
 
 - feat: Add deletion of stack after destroy (remove flag)
+- feat: Add support for an `install` command, similar to
+  [setup-pulumi](https://github.com/marketplace/actions/setup-pulumi)
```

Comment:
Why is this listed under 3.2.1 and not HEAD?

Reply:
Ah, probably because there was a release since this change was made and git merge decided to put it in the old position. Good catch.